api/api/alogging.py

```diff
@@ -2,15 +2,13 @@
 # Created by Wazuh, Inc. <info@wazuh.com>.
 # This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
 
-import collections
 import logging
 import json
 import re
-from pythonjsonlogger import jsonlogger

```

Comment:
We need to remove this from the requirements https://github.com/wazuh/wazuh/blob/1f351bba9968c5a0557ea9fddea159ce26b3471d/framework/requirements.txt#L22

Reply:
You are right, the embedded interpreter downloads the dependencies at installation time.